logos/gigalayer/gigalayer.svg

```diff
@@ -0,0 +1,103 @@
+<?xml version="1.0" encoding="utf-8"?>

```

Comment:
Hi @ladaposamuel, thank you for taking time to contribute to this. 

Please can you run your files through [SVGOMG](https://jakearchibald.github.io/svgomg/) and an image optimizer, as recommended in the [contributing.md](https://github.com/PaystackHQ/nigerialogos/blob/master/contributing.md) file.

Reply:
I have done this thanks.